azurerm/internal/services/storage/storage_management_policy_resource.go

```diff
@@ -81,6 +84,35 @@ func resourceStorageManagementPolicy() *schema.Resource {
 										},
 										Set: schema.HashString,
 									},
+
+									"blob_index_match": {
```

Comment:
If we change this to
```suggestion
									"blob_index_match_tag": {
```